.travis.yml

```diff
@@ -43,28 +44,31 @@ install:
   - CYTHON_COVERAGE=1 pip install -e .
 
 script:
-  - if [[ "${BUILD_DOCS}" == "true" ]]; then
+  - if [[ $TRAVIS_JOB_NAME == python-* ]]; then
+      pytest ;
```

Comment:
Ideally we should run the tests outside of the build folder. If you are OK with the changes in this commit I'll send a new one to do that too.